spec/interactors/content/update_interactor_spec.rb

```diff
@@ -3,57 +3,63 @@
 RSpec.describe Content::UpdateInteractor do
   describe ".call" do
     before { stub_any_publishing_api_put_content }
-    let(:edition) { create(:edition) }
+    let(:edition) { create(:edition, number: 2) }
     let(:user) { build(:user) }
 
-    def build_params(document: edition.document,
-                     title: SecureRandom.alphanumeric(10),
-                     summary: SecureRandom.alphanumeric(10))
-      ActionController::Parameters.new(
-        document: document.to_param,
-        revision: { title: title, summary: summary },
-      )
+    let(:params) do
+      ActionController::Parameters.new(document: edition.document.to_param,
+                                       title: "New title",
+                                       summary: "New summary",
+                                       change_note: "New note")
     end
 
     it "succeeds with default parameters" do
-      result = Content::UpdateInteractor.call(params: build_params, user: user)
+      result = Content::UpdateInteractor.call(params: params, user: user)
       expect(result).to be_success
     end
 
     it "updates the edition" do
-      params = build_params(title: "New title", summary: "New summary")
-
       expect { Content::UpdateInteractor.call(params: params, user: user) }
         .to change { edition.reload.title }.to("New title")
         .and change { edition.reload.summary }.to("New summary")
+        .and change { edition.reload.change_note }.to("New note")
+    end
+
+    it "ignores change notes for first editions" do
+      params.merge!(document: (create :edition).document.to_param)
+
+      expect { Content::UpdateInteractor.call(params: params, user: user) }
+        .to_not(change { edition.reload.change_note })
     end
 
     it "creates a timeline entry" do
-      expect { Content::UpdateInteractor.call(params: build_params, user: user) }
+      expect { Content::UpdateInteractor.call(params: params, user: user) }
         .to change { TimelineEntry.where(entry_type: :updated_content).count }
         .by(1)
     end
 
     it "updates the preview" do
       expect(FailsafeDraftPreviewService).to receive(:call).with(edition)
-      Content::UpdateInteractor.call(params: build_params, user: user)
+      Content::UpdateInteractor.call(params: params, user: user)
     end
 
     it "raises an error when the edition isn't editable" do
-      params = build_params(document: create(:edition, :published).document)
+      params.merge!(document: (create :edition, :published).document.to_param)
```

Comment:
This use of parenthesis seems somewhat unconventional of wrapping the full method call in parenthesis rather than the arguments.

Reply:
Oh yeah - I see what you mean. Updated 👍